awx/api/generics.py

```diff
@@ -882,6 +883,21 @@ def copy_model_obj(old_parent, new_parent, model, obj, creater, copy_name='', cr
                 create_kwargs[field.name] = CopyAPIView._decrypt_model_field_if_needed(
                     obj, field.name, field_val
                 )
+
+        # WorkflowJobTemplateNodes that represent an approval are *special*;
+        # when we copy them, we actually want to *copy* the UJT they point at
+        # rather than share the template reference between nodes in disparate
+        # workflows
```

Comment:
I get it.

> we actually want to *copy* the UJT they point at rather than share the template reference between nodes in disparate workflows

little confused by wording "share the template reference", I might word it "copying the reference to the template".

Our intent is that Approval JTs are 1-to-1 with their WFJT node. This special case is demanded by that requirement. Good job hunting that down.